contracts/modules/RelayerManager.sol

```diff
@@ -268,19 +254,12 @@ abstract contract RelayerManager is BaseModule {
     * The method MUST throw if one or more signatures are not valid.
     * @param _wallet The target wallet.
     * @param _signHash The signed hash representing the relayed transaction.
-    * @param _signatures The signatures as a concatenated byte array.
-    * @param _option An enum indicating whether the owner is required, optional or disallowed.
+    * @param _signatures The signatures as a concatenated bytes array.
+    * @param _option An OwnerSignature enum indicating whether the owner is required, optional or disallowed.
     * @return A boolean indicating whether the signatures are valid.
     */
-    function validateSignatures(
-        address _wallet,
-        bytes32 _signHash,
-        bytes memory _signatures,
-        OwnerSignature _option
-    )
-        internal
-        view
-        returns (bool)
+    function validateSignatures(address _wallet, bytes32 _signHash, bytes memory _signatures, OwnerSignature _option) internal view
```

Comment:
Why the new format for long method definitions? Can we stick to the current convention which we use everywhere in the code?

Reply:
That is the convention we agreed a year ago but isn't applied here to use 145 chars and/or 5 parameters on single line and only drop to multi line otherwise.

Reply:
As per the discussion we agreed to "... have either everything on the same line, of drop everything on their own lines. But I don't like dropping only the modifiers.".

So to me me explicitly agreed not to use the above convention.

Reply:
I've changed them to one line then.
Unfortunately I can't find a solhint rule for that so we can apply it consistently.

Reply:
Sorry to insist but can we have everything in one line then? The `returns` is still in a separate line for several methods (all the `multiCall`s and `validateSignatures`) which is not consistent and harder to read.